run_tests.py

```diff
@@ -1,17 +1,21 @@
 # -*- coding: utf-8 -*-
-from flake8_builtins import BuiltinsChecker
-
 import ast
 import unittest
 
+from flake8_builtins import BuiltinsChecker
+
```

Comment:
Please revert this change and the one above, i.e. keep the imports as they were.